.changeset/friendly-poems-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `selector-pseudo-class-no-unknown` resource state false positives
```

Comment:
Oops, I missed it. 👍🏼 A trailing whitespace may be removed:

```suggestion
Fixed: `selector-pseudo-class-no-unknown` false positives for resource state
```

Reply:
I went for something more accurate:
```
Fixed: `selector-pseudo-class-no-unknown` false positives for `:seeking`, the media loading state and sound state pseudo-classes
```

Reply:
Hum, but not only `:seeking`, right? 🤔

Reply:
see
https://www.w3.org/TR/selectors-4/#media-loading-state
https://www.w3.org/TR/selectors-4/#sound-state
web-platform-tests/interop#224

Reply:
Does just removing `:seeking` make sense instead?

```
for the media loading state and sound state pseudo-classes
```

Reply:
Nop because `:seeking` is part media playback state pseudo-classes.
We had 2 of them—`:playing` and `:paused`—already supported but `:seeking` was missing.

Reply:
Ah, make sense. 👍🏼